### PR TITLE
Arquillian Cube 1.2.0 and 1.3.0 announcements

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

File: blog/arquillian-cube-1.2.0.textile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+date: 2017-05-11
```
Comment (bartoszmajsak) [inline blog/arquillian-cube-1.2.0.textile]: Maybe we can align dates with release dates?

Comment (lordofthejars) [inline blog/arquillian-cube-1.2.0.textile]: my idea since it was so long ago was to just order and appear at top of the web site

Comment (bartoszmajsak) [inline blog/arquillian-cube-1.2.0.textile]: I see your point. Let's see what else do we have in the line and maybe I will adjust the dates

**Comment by MatousJobanek** [comment]
Landed in:
https://github.com/arquillian/arquillian.github.io/commit/11bbee5985f6098d396b418429f1de6edaa9fb46
https://github.com/arquillian/arquillian.github.io/commit/d45ca7bc7ef13ae6aa717fe697ea09065312a6a6

**Pull request closed without merging**